package.json

```diff
@@ -2,54 +2,54 @@
   "name": "ember-awesome-macros",
   "version": "0.25.0",
   "description": "A random collection of Ember computed macros",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "Kelly Selden",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kellyselden/ember-awesome-macros.git"
+  },
   "scripts": {
     "build": "ember build",
     "start": "ember server",
     "test": "ember try:each"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/kellyselden/ember-awesome-macros.git"
-  },
-  "engines": {
-    "node": ">= 0.12.0"
+  "dependencies": {
+    "babel-plugin-fake-import-specifiers": "^1.0.0",
+    "ember-cli-babel": "^5.1.7",
+    "ember-macro-helpers": "0.2.0"
   },
-  "author": "Kelly Selden",
-  "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
     "ember-ajax": "^2.4.1",
-    "ember-cli": "2.9.1",
+    "ember-cli": "2.10.0",
     "ember-cli-app-version": "^2.0.0",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-htmlbars": "^1.0.10",
     "ember-cli-htmlbars-inline-precompile": "^0.3.3",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-jshint": "^2.0.0",
+    "ember-cli-jshint": "^2.0.1",
     "ember-cli-qunit": "^3.0.1",
     "ember-cli-release": "^0.2.9",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-data": "^2.9.0",
+    "ember-data": "^2.10.0",
```

Comment:
Yes, some of the promise macros use mixins from it. But I'm totally open to extracting those macros out to remove the ember-data dependency.